src/kEvent.ml

```diff
@@ -690,6 +879,15 @@ let proved_xml mdl level trans_sys k prop_name =
          | None -> ()
          | Some msg -> Format.fprintf ppf " comment=\"%s\"" msg
       )
+      (match prop.prop_kind with
+      | Property.Invariant -> "valid"
+      | Property.Reachable Some (From ts) -> "unreachable from " ^ string_of_int ts ^ " timesteps"
+      | Property.Reachable Some (Within ts) -> "unreachable within " ^ string_of_int ts ^ " timesteps"
+      | Property.Reachable Some (At ts) -> "unreachable at " ^ string_of_int ts ^ " timesteps"
+      | Property.Reachable Some (FromWithin (ts1, ts2)) -> "unreachable from " ^ string_of_int ts1 ^ " and within " ^ string_of_int ts2 ^ " timesteps"
+      | Property.Reachable None -> "unreachable"
```

Comment:
Don't use verbose values for XML/JSON. Just use `"unreachable"`.  XML/JSON values should be easily parseable. If we decide to provide info about the bounds (in addition to the result) later, we should do it in a more structured way.